### DISCUSSION TITLE
vocore2

### DISCUSSION THREAD

**Discussion by sp00ck** [markqvist/Reticulum#220] [Ideas]
If python is necessary meybe glue a https://vocore.io/v2.html and any lora dev in one?

No need keyboard, screen etc. All communication is possible trought normal www server. 
every device with web browser or telnet can be used.

In the current state of the system, I am against all esp and small STM computers.
Normal Linux with mainline kernel. And preferably wifi mesh. Wifi alone has great potential but no one has done networking with only on wifi

**Comment by buhalo1** [comment]
An interesting solution. Currently, I am building an outdoor node with a power of 1 W, powered by solar panels, due to the inability to use only esp32, as is done in meshtastic, in addition to a self-made rnode based on wroom32 and a highpower semtech transceiver, I additionally have to use an rpi as part of the node, which is sgnificantly increases the requirements for the power supply and cooling/heating system, and significantly complicates the design, up to the need to use a cooling system based on Peltier elements.
Using only a vocore and a semtech transceiver would greatly simplify node design.

**Comment by markqvist** [comment]
The VoCore2 should be sufficient, yes. Also a Pi Zero works fine (although currently unobtanium).
